supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/Run.scala

```diff
@@ -117,7 +117,20 @@ case class Run(runId: String,  jobDescriptor: BackendJobDescriptor, genomicsInte
       entry <- events.asInstanceOf[JArrayList[GArrayMap[String, String]]].asScala
     } yield EventStartTime(entry.get("description"), OffsetDateTime.parse(entry.get("startTime")))
 
-    val finaleEvents: Seq[EventStartTime] = eventIfExists("endTime", metadata, "cromwell poll interval").toSeq
+    // A little bit ugly... the endTime of the jes operation can actually be before the final "event" time, due to differences
+    // in the reported precision. As a result, we have to make sure it all lines up nicely:
+    val finaleEvents: Seq[EventStartTime] = {
```

Comment:
Brought forward from before (but since I also originally wrote this, I'll claim that yes, it's an artistic flourish...)